content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.getTimeInMillis()'
+Description: 'Returns the time in milliseconds.'
+Subjects: 
+	- 'Code Foundations'
+	- 'Computer Science'
+Tags:
+	- 'Date'
+	- 'Methods'
+CatalogContent:
+	- 'learn-java'
+	- 'paths/computer-science'
+---
+
+The **`.getTimeInMillis()`** method of the `Calendar` class returns the time represented by a calendar object in the form of milliseconds.
+
+## Syntax
+
+```pseudo
+calendar.getTimeInMillis()
+```
+
```

Comment:
There should be some additional information here, such as what is the expected return value